pages/category.html

```diff
@@ -0,0 +1,114 @@
+
+      <div class="cards-container">
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify bopis image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Bopis</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify preorder image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Preorder</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify hc oms image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Hc Oms</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+
+        <div class="cards">
+          <div>
+            <a href="">
+              <div>
+                <!--TODO insert shopify store mode image-->
+                <img src="../img/HC Logo.svg" class="logo">
+              </div>
+              <div>
+                <h3>Shopify Store Mode</h3>
+                <p>HotWax Commerce provides retailers with the best return on their inventory by facilitating omnichannel retailing strategies such as same-day Buy Online Pick-Up In Store (BOPIS), Ship From Store, and Pre-Orders in order to increase conversion rates and profitability.</p>
+              </div>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
+
+<style>
+a {
+  text-decoration: none;
+}
+
+.logo {
+  width: auto;
+  height: 58px;
+}
+
+.banner-content,
+.card-header {
+  text-align: center;
+}
+
+.cards-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(275px, 1fr));
```

Comment:
this grid will be for desktop, on mobile we want the cards to be just one after another vertically similar to the concept we use in the BOPIS fulfillment app. optimize this styling to be mobile first.

Reply:
Done